internal/database/database_container.go

```
@@ -0,0 +1,388 @@
+// Build uses the information stored in the builder to create the database instance.
+func (b *InstanceBuilder) Build() (result *Instance, err error) {
+	s := b.server
+
+	// Generate the database name and password:
+	name := fmt.Sprintf("test_%d", s.count)
+	user := fmt.Sprintf("test_%d", s.count)
+	password := uuid.NewString()
+	s.count++
+
+	// Create the user:
+	_, execErr := s.handle.Exec(fmt.Sprintf(
```

golangci-lint (2.12.2)

[error] (*database/sql.DB).Exec must not be called. use (*database/sql.DB).ExecContext

(noctx)